### DISCUSSION TITLE
Initial state of editor when new canvas is loaded from remote storage with data: null

### DISCUSSION THREAD

**Discussion by contentfree** [GrapesJS/grapesjs#4486] [Q&A]
I have a process that creates a new template in our backend before showing the Grapesjs editor for that template. I'm using remote storage. It seems that on the initial load from storage manager the result is an empty gjs-frames element and no canvas is shown. This is almost certainly because the data returned of the brand-new template is null.

How do I either A) prevent null data from wiping out the current canvas (built from an element on initialization of the editor) or B) set up my "blank slate" data when I create the template in the backend?

**Comment by artf** [comment]
Hi @contentfree here you can see an example of an empty project data https://grapesjs.com/docs/modules/Storage.html#setup-remote-storage
```json
{"assets": [], "styles": [], "pages": [{"component": "<div>Initial content</div>"}]}
```
You should also be able to skip the reset of the current data by returning an empty project data `{}`
